tests/src/system/packages/MessagingFeedTests.scala

```diff
@@ -0,0 +1,190 @@
+        val iter = kafka_brokers_sasl_json_array.iterator();
+        while(iter.hasNext()){
+            val server = iter.next().getAsString()
+            vec = vec :+ JsString(server)
+            servers = s"$servers$server,"
+        }
+        var kafka_brokers_sasl = JsArray(vec)
+   
+        // Create a temporary configuration file for Java security.
+        val data = List("KafkaClient {",
+            "com.ibm.messagehub.login.MessageHubLoginModule required",
+            "serviceName=\"kafka\"", "username=\"" + user + "\"",
+            "password=\"" + password + "\";",
+            "};")
+        val currentTime = s"${System.currentTimeMillis}"
+        val tempConfigFile = s"jaas_$currentTime.conf"
+        val writer = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(tempConfigFile)))
+        for (x <- data) {
+            writer.write(x + "\n")
+        }
+        writer.close()
+
+        System.setProperty("java.security.auth.login.config", tempConfigFile)
+        var props = new Properties()
+        props.put("bootstrap.servers", servers);
+        props.put("security.protocol", "SASL_SSL");
+        props.put("key.serializer", "org.apache.kafka.common.serialization.StringSerializer");
+        props.put("value.serializer", "org.apache.kafka.common.serialization.StringSerializer");
+ 
+        (wp, assetHelper) =>
+            val triggerName = s"dummyMessageHubTrigger-$currentTime"
+
+            // Make sure the package of the Kafka feed exists.
+            val packageGetResult = wsk.pkg.get(messagingPackage)
+            packageGetResult.stdout should include("ok")
+  
+            // Create a trigger with the message hub feed.
+            val feedCreationResult = assetHelper.withCleaner(wsk.trigger, triggerName) {
+                (trigger, name) =>
+                    trigger.create(name, feed = Some(s"$messagingPackage/$messageHubFeed"), parameters = Map(
+                        "user" -> user.toJson,
+                        "password" -> password.toJson,
+                        "api_key" -> api_key.toJson,
+                        "kafka_admin_url" -> kafka_admin_url.toJson,
+                        "kafka_brokers_sasl" -> kafka_brokers_sasl,
+                        "topic" -> topic.toJson))
+            }
+            feedCreationResult.stdout should include("ok")
+
+            // It takes a moment for the consumer to fully initialize.
+            Thread.sleep(2000)
+            
+            // Send a message to the message hub service.
+            val producer = new KafkaProducer[String, String](props)
+            val record = new ProducerRecord(topic, "key", currentTime)
+            producer.send(record)
+            producer.close()
+            
+            // Check if the trigger is fired.
+            val activations = wsk.activation.pollFor(N = 1, Some(triggerName), retries = 30).length
+            withClue("Change feed trigger count: ") { activations should be(1) }
```

Comment:
You should additionally check that the content of the activation contains the message you posted, as well as the other expected fields: topic, partition, offset, etc.